### PR TITLE
feat: footer March

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This PR awaits Abbey's confirmation for replacing a 28+ character footer item with a shorter version.

### CODE REVIEW

**Comment by ieahleen** [comment]
could you please replicate this change in `italian/trending.json`? we don't yet have enough articles for updating it with italian articles

**Comment by ahmadabdolsaheb** [comment]
Sure, let's discuss this further to see if we need to implement this strategy across languages and across platforms.

File: client/i18n/locales/english/trending.json [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
+  "article24link": "https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/",
+  "article25title": "What is Graphic Design?",
+  "article25link": "https://www.freecodecamp.org/news/what-is-graphic-design-the-graphic-designer-job-explained-in-plain-english/",
+  "article26title": "Share Screen on Android",
+  "article26link": "https://www.freecodecamp.org/news/share-screen-on-android-how-to-cast-my-screen-with-a-windows-10-pc/",
+  "article27title": "Problem Resetting Your PC",
+  "article27link": "https://www.freecodecamp.org/news/there-was-a-problem-with-resetting-your-pc-reset-pc-fixed-on-windows-10/",
+  "article28title": "Open Task Manager Windows 10",
+  "article28link": "https://www.freecodecamp.org/news/how-to-open-task-manager-in-windows-10/",
+  "article29title": "How to Delete Albums on iPhone",
```
Comment (ahmadabdolsaheb) [inline client/i18n/locales/english/trending.json:60]:
```suggestion
  "article29title": "Delete Albums on iPhone",
```

**Review by Sembauke — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍

**Pull request merged**